src/app/waiting/page.tsx

```diff
@@ -20,54 +30,81 @@ export default function WaitingView() {
       copyRef.current.click();
     }
   };
+  useEffect(() => {
+    const storedToken = window.sessionStorage.getItem('accessToken') || '';
 
-  return (
-    <div className="flex w-full flex-col items-center gap-y-8">
-      <div className="text-center text-xl">
-        프로필 생성 중 <br />
-        조금만 기다려 주세요!
-      </div>
+    if (storedToken !== '') {
+      fetch(
+        `${process.env.NEXT_PUBLIC_CLIENT_ADDRESS}:${process.env.NEXT_PUBLIC_CLIENT_PORT}/api/status?code=${storedToken}`,
+      )
+        .then((response) => response.json())
+        .then(async (data) => {
+          if (data.code === 200) {
+            setPhotoMade(data.data);
+          } else {
+            setErrorMessage(STATUS_MSG);
+          }
+        })
+        .catch(() => {
+          setErrorMessage(STATUS_ERROR_MSG);
+        });
+    }
+  }, [setErrorCheckMessage, setErrorMessage]);
 
-      <Image
-        src="/unlocked-profile.png"
-        alt="Sample Image"
-        width={205}
-        height={205}
-        priority
-      />
+  useEffect(() => {
+    if (photoMade !== null) {
+      router.push('/frame');
```

Comment:
```suggestion
router.push(ROUTE_TYPES.FRAME);
```
`src/app/frame/page.tsx`의 43번째 줄 처럼 
해당 페이지도 routing할때 주소를 잘못 입력하는 실수를 방지하기 위해 상수를 사용하시는 것을 권장드립니다

Reply:
이런,, 정말 꼼꼼하시군요
감사합니다~~